testsuite/features/step_definitions/common_steps.rb

```diff
@@ -42,15 +42,18 @@
 
 Then(/^the IPv4 address for "([^"]*)" should be correct$/) do |host|
   node = get_target(host)
+  puts node.public_ip
```

Comment:
I'm assuming this was for debugging and is not needed for the tests? Or should we leave it anyway?

Reply:
no, it's intended to stay. we sometimes output some minimal information on how the tests proceeded